src/test/java/org/apache/bcel/PLSETestCase.java

```diff
@@ -116,6 +119,25 @@ public void testB295() throws Exception
         assertEquals(lv.getLength(), new_lv.getLength(), "live range length");
     }
 
+    /**
+     * BCEL-361: LineNumber.toString() treats code offset as signed
+     */
+    @Test
+    public void testB361() throws Exception
+    {
+        final JavaClass clazz = getTestClass(PACKAGE_BASE_NAME+".data.LargeMethod");
```

Comment:
Missing spaces around `+`. I'm on vacation but there is a test I want to run... back in a while...